### DISCUSSION TITLE
Work on Fedora or RHEL distros?

### DISCUSSION THREAD

**Discussion by psi4j** [gokcehan/lf#1021] [General]
Does this have some sort of workaround for Fedora 37 or other RHEL-based distros?

**Comment by petraglyph** [comment]
What do you mean by "workaround'? You can install lf on Fedora 37 with Copr (see [wiki](https://github.com/gokcehan/lf/wiki/Packages#fedora)) if that's what you're looking for.

**Reply by psi4j** [reply]
Thank you. The golang Unix method doesn't work for some reason, but that worked great.
